configurator/src/cv.rs

```diff
@@ -28,7 +26,7 @@ impl Error for CameraError {}
 
 #[derive(Default)]
 pub struct Picture {
-    inner: Mat,
+    pub inner: Mat,
 }
```

Comment:
I feel like maybe a better solution would be to make a `save_file` method on the `Picture` and use that in the other file. Less code duplication as well.

Reply:
Done